src/app/Main/Body/QueryPage/QueryPage-Container.component.tsx

```diff
@@ -36,6 +36,13 @@ export function QueryPageContainerComponent(): JSX.Element {
         const filterResponse: Response = await fetch(FILTER_URL);
         const filterProp: FilterConfigApiInterface = await filterResponse.json();
 
+        filterProp.filters.forEach((element, index) => {
+            if (element.id === "sContext") {
+                filterProp.filters[index].id = "samplingContext";
+                filterProp.filters[index].name = "samplingContext";
+            }
+        });

```

Comment:
Reject: It is in general a bad idea to manipulating the data structure you are iterating over. I would suggest to map the filterProps to the desired datastructure. So const newFilterProps =  filterProp.filters.map(...)...

Reply:
See comment in previous commit: Here you already mapping the api data to a more appropriate structure fitting your components desire. So you should use separate models for api and client data.

Reply:
done. 
- 1. Reject in "Only use desired values from isolate-API"
- 2.Comment in "Separate api_isolate_model from data_isolate_model"